compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -266,9 +266,14 @@ void OMR::ValuePropagation::initialize()
    _blocksToBeRemoved = new (trStackMemory()) TR_Array<TR::CFGNode*>(trMemory(), 8, false, stackAlloc);
    _curDefinedOnAllPaths = NULL;
    if (_isGlobalPropagation)
+      {
       _definedOnAllPaths = new (trStackMemory()) DefinedOnAllPathsMap(std::less<TR::CFGEdge *>(), trMemory()->currentStackRegion());
+      _callNodeToGuardNodes = new (trStackMemory()) CallNodeToGuardNodesMap(std::less<TR::Node *>(), trMemory()->currentStackRegion());
+      }
    else
+      {
       _definedOnAllPaths = NULL;
```

Comment:
Should we also null out `_callNodeToGuardNodes` here?

Reply:
It will not be used but let me do it for symmetry

Reply:
Done